UnitsNet/Scripts/UnitDefinitions/Pressure.json

```diff
@@ -8,7 +8,7 @@
             "PluralName": "Pascals",
             "FromUnitToBaseFunc": "x",
             "FromBaseToUnitFunc": "x",
-            "Prefixes": ["Kilo","Mega"],
+            "Prefixes": ["Micro","Deca","Hecto","Kilo","Mega","Giga"],
```

Comment:
Of course it's possible to convert it to Milli and Centi but I never see it in the wild. In my company we we are working lot with measuring of preasure. There I also never see a device that use MilliPascal or CentiPascal.
As source in the internet I only find a german wiki entry where you can also find these prefixes: http://de.wikipedia.org/wiki/Pascal_(Einheit)